app/models/wallet.rb

```diff
@@ -21,6 +21,8 @@ def mark_as_terminated!(timestamp = Time.zone.now)
     terminated!
   end
 
+  scope :expired, -> { where('wallets.expiration_date < ?', Time.zone.now.beginning_of_day,) }
```

Comment:
```suggestion
  scope :expired, -> { where('wallets.expiration_date < ?', Time.current.beginning_of_day,) }
```

Reply:
Fixed, actually wasn't aware before`.current` covers it :)